src/app/app.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+var winnersString = {
```

Comment:
Cos è questa cosa qui?
- Se un giorno avremo 1000 traduzioni o pluralizzazioni pensi sia un buon punto dove mettere?
- In questo modo stai definendo una variabile globale. Ti sembra una buona strategia?